src/mobu/services/business/nublado.py

```diff
@@ -409,8 +339,15 @@ async def setup_session(self, session: JupyterLabSession) -> None:
             reference=reference.strip() if reference else None,
             description=description.strip() if description else None,
         )
+        set_tag(
+            "image",
+            self._image.description
+            or self._image.reference
+            or "<image unknown>",
+        )
```

Comment:
These are separate things. I would instead set separate tags for the image description and the image reference.